frontend/js/src/user/playlists/Playlists.tsx

```diff
@@ -23,6 +23,7 @@ import CreateOrEditPlaylistModal from "../../playlists/components/CreateOrEditPl
 import ImportPlaylistModal from "../../playlists/components/ImportPlaylistModal";
 import PlaylistsList from "./components/PlaylistsList";
 import { getPlaylistId, PlaylistType } from "../../playlists/utils";
+import ImportPlaylistModal from "./components/ImportPlaylistModal";
```

Comment:
There's a duplicate import of `ImportPlaylistModal` above that should be removed (or this line)

Reply:
Neeever mind, I realize this is due to a small conflict with #2805 that adds another button for importing playlist JSPF files.

I hope you don't mind, I'm going to resolve the issue now because otherwise I can't build the image to deploy it and test,